wp-content/themes/workday/less/widgets.less

```diff
@@ -142,3 +142,88 @@
     }
   }
 }
+
+.home #widget-area .largo-recent-posts {
```

Comment:
If these styles will only ever be applied on the homepage, let's remove the `.home` selector and put it in `/homepages/assets/less/homepage.less`, which is only loaded on the homepage.